core/src/text/fontContext.cpp

```diff
@@ -185,6 +185,11 @@ bool FontContext::layoutText(TextStyle::Parameters& _params, const std::string&
                      (metrics.aabb.y + height * 0.5) * TextVertex::position_scale);
 
     auto it = _quads.begin() + quadsStart;
+    if (it == _quads.end()) {
```

Comment:
Why not just check for size of `_quads` at the beginning before doing any textWrapping stuff?